website/integrations/services/actual-budget/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Integrate with Actual Budget
+sidebar_label: Actual Budget
+---
+
+# Actual Budget
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Actual Budget
+
+> Actual Budget is a web-based financial management software. It helps users track and manage their income, expenses, and budgets in real time.
+> The software compares actual spending with planned budgets to improve financial decisions.
+>
+> -- https://actualbudget.org/
+>
+> This guide explains how to configure Actual Budget to use authentik as the OAuth provider for logging in to the Web GUI.
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `actual.company` is the FQDN of the Actual Budget install.
+- `authentik.company` is the FQDN of the authentik install.
+
+## authentik configuration
+
+[Create](https://docs.goauthentik.io/docs/add-secure-apps/applications/manage_apps#add-new-applications) an OAuth2/OpenID provider and an application in authentik. Use the following parameters for the OAuth2/OpenID provider:
+
+**Provider:**
+
+- Name: `SP-actual`
+- Client type: `Confidential`
+- Redirect URIs/Origins (RegEx): `https://actual.company/openid/callback`
+- Signing Key: Select any available signing keys.
+
+:::info
+Actual Budget supports the RS256 algorithm. Be aware of this when choosing the appropriate signing key.
+:::
+
+Take note of the Client ID and Client Secret; you will need to provide them to Actual Budget in the last step.
+
+Leave the remaining values as default. Durations can be adjusted as needed.
+
+**Application:**
+
+- Name: `Actual Budget`
+- Slug: `actual`
+- Launch URL: `https://actual.company/`
+
+## Actual Budget configuration
+
+1. Sign in to Actual Budget with a browser of your choice and access your budget by clicking on its name.
+
+2. Click your budget in the top-left corner to open the dropdown menu and select `Settings`.
```

Comment:
```suggestion
2. Click your budget in the top-left corner to open the dropdown menu and select **Settings**.
```

Reply:
Yes, now I see what 4d62 was saying.... the UI components, the buttons and field names, etc., should be bolded, not in code format.